libraries/provision/ansible/playbooks/fetch-windows-cbl-logs.yml

```diff
@@ -0,0 +1,11 @@
+- name: Windows - Stop TestServer
+  hosts: windows
+  vars:
+    log_full_path:
+
+  tasks:
+    - include: tasks/stop-fetch-windows-cbl-logs.yml
+
+    - name: Fetch logs
+      fetch: src=C:\PROGRA~1\dotnet\TextLogging\{{ item }} dest={{ log_full_path }} fail_on_missing=yes flat=yes validate_checksum=no
```

Comment:
This is already being done by stop-fetch-windows-cbl-logs.yml. This yml is probably not necessary?

Reply:
done, removed the fetching task from fetch-windows-cbl-logs.yml